modules/components/pages/caches/nats_kv.adoc

```diff
@@ -164,16 +166,24 @@ Whether to allow the remote server to repeatedly request renegotiation. Enable t
 *Type*: `bool`
 
 *Default*: `false`
+
+ifndef::env-cloud[]
 Requires version 3.45.0 or newer
+endif::[]
 
 === `tls.root_cas`
 
 An optional root certificate authority to use. This is a string, representing a certificate chain from the parent trusted root certificate, to possible intermediate signing certificates, to the host certificate.
+
+ifndef::env-cloud[]
```

Comment:
Would be nice to single-source this too.

Reply:
Done

Reply:
you could probably have a single file with this content to avoid repeating all these `ifdef`

```
ifndef::env-cloud[]

[CAUTION]
====
This field contains sensitive information that usually shouldn't be added to a configuration directly, read our xref:configuration:secrets.adoc[secrets page for more info].
====

endif::[]
ifdef::env-cloud[]

[CAUTION]
====
This field contains sensitive information. Review your xref:redpanda-cloud:security:authorization/cloud-authorization.adoc[cluster security] before adding it to your configuration.

====endif::[]
```

Reply:
this has the added benefit of writers not missing an update for either platform. The content for both lives in the same place.

Reply:
Done